src/components/Sidebar.tsx

```diff
@@ -46,12 +48,17 @@ const Sidebar = ({ route }: Props) => {
                 key={key}
                 href={key}
                 className={twMerge(
-                  'p-2 rounded-md border border-gray-200 border-opacity-20 shadow-md',
+                  'flex flex-row p-2 rounded-md border border-gray-200 border-opacity-20 shadow-md',
                   key === route
                     ? 'bg-gray-400 bg-opacity-50 text-indigo-600 hover:bg-opacity-90'
                     : 'hover:bg-gray-400 hover:bg-opacity-50 hover:text-indigo-600 ',
                 )}
               >
+                {key === Routes.DASHBOARD ? (
```

Comment:
Change mapping type to have a name and an icon and use the icon from the values instead